packages/awslint/README.md

```diff
@@ -90,6 +90,21 @@ Examples:
 When a rule is excluded, it will be displayed as `skipped:` in the output and
 will always considered to pass.
 
+## Flags
```

Comment:
Feels a bit heavy handed for a temporary situation (it should take a couple of days to migrate the repo and then we can just remove all this complexity).

Maybe just a simple temporary environment variable??

Reply:
It takes the time pressure (couple of days) away by making this formal. We may come across a module that cannot be migrated and we may end up leaving it as is (accepting the cost of merge conflicts).

Also, don't forget that `awslint` is a published package that some of our customers are using. We want this mechanism to be available to our customers on their own timeline as they also go through the motions of adopting the "constructs" module. So why not make it formal?

Perhaps I should mark this flag as deprecated to indicate that it will not be present in `awslint-2.0`.

Reply:
Even if it's a simple env var, we'll need to include a mechanism to configure this in `package.json`, no?

Reply:
Sure but that's something we can easily inject in our build tools. The fact that awslint is public makes this complexity even less appealing given there's a single use case here that's not even long-lived.

I am concerned that this is just over designing something that's supported to be a one-off. I am not aware or can think of future use cases.

Reply:
So the env bar solution will look something like this -

The `cdk-build` section within `package.json` will contain an `env` key. The environment variables set here will be passed down to all child processes of cdk-build. 

An environment variable will be set in this key for packages that have migrated to use the "constructs" module. This will also be recognized by `awslint` and behave differently.

Does this align with your suggestion?

Note that, by default, invoking the `cdk-build` and `awslint` binaries for these packages will result in different behaviours.

Something about the differing behaviours between the binaries feels odd and hacky.

Reply:
Yes sounds about right. I agree it's not ideal but since this is really a one off workaround for the transitional period I don't think that's an issue really.

Alternatively, you could add the env var inside the package.json `awslint` script:

```json
{
  "scripts": {
    "awslint": "AWSLINT_BASE_CONSTRUCT=1 awslint"
  }
}
```

This will allow users to continue to run `yarn awslint`.

Reply:
Changed. Take another look.